packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -180,24 +191,75 @@ describe('c-quantic-search-box-input', () => {
       });
 
       describe('when the suggestions list is not empty', () => {
-        it('should display the suggestions in the suggestions list', async () => {
-          const element = createTestComponent({
-            ...defaultOptions,
-            suggestions: mockSuggestions,
-            textarea: textareaValue,
+        describe('when only query suggestions are displayed', () => {
+          it('should display the suggestions in the suggestions list', async () => {
+            const element = createTestComponent({
+              ...defaultOptions,
+              suggestions: mockSuggestions,
+              textarea: textareaValue,
+            });
+            await flushPromises();
+
+            const input = element.shadowRoot.querySelector(
+              textareaValue
+                ? selectors.searchBoxTextArea
+                : selectors.searchBoxInput
+            );
+            expect(input).not.toBeNull();
+            await input.focus();
+
+            const suggestionsList = element.shadowRoot.querySelector(
+              selectors.searchBoxSuggestionsList
+            );
+            expect(suggestionsList).not.toBeNull();
+
+            const suggestionsListItems =
+              suggestionsList.shadowRoot.querySelectorAll(
+                selectors.suggestionOption
+              );
+            expect(suggestionsListItems).not.toBeNull();
+            expect(suggestionsListItems.length).toEqual(mockSuggestions.length);
           });
-          await flushPromises();
+        });
 
-          const suggestionsList = element.shadowRoot.querySelector(
-            selectors.searchBoxSuggestionsList
-          );
-          expect(suggestionsList).not.toBeNull();
+        describe('with both query suggestions and recent queries available', () => {
```

Comment:
This is where you could also have a unit test to test that if you have the same query in recent and in query suggest, it's not displayed twice

Reply:
Indeed, this will be part of this: https://coveord.atlassian.net/browse/SFINT-5645